### PR TITLE
Update otlp-exporter.md

### PR DESCRIPTION
**Pull request by shalk**

### CODE REVIEW

File: content/en/docs/languages/sdk-configuration/otlp-exporter.md [hunk 1]
```diff
@@ -162,10 +162,13 @@ The timeout value for all outgoing logs in milliseconds.
 
 **Example:** `export OTEL_EXPORTER_OTLP_LOGS_TIMEOUT=500`
 
-### `OTEL_EXPORTER_OTLP_PROTOCOL`
-
 ## Protocol Configuration
 
+The following environment variables configure the OTLP transport protocol
+an OTLP Exporter will use HTTP or gRPC protocol to send requests.
```
Comment (cartermp) [inline content/en/docs/languages/sdk-configuration/otlp-exporter.md:168]:
```suggestion
The following environment variables configure the OTLP transport protocol
an OTLP Exporter uses.
```

**Review by cartermp — COMMENTED** [review]
drive-by suggestion

File: content/en/docs/languages/sdk-configuration/otlp-exporter.md [hunk 2]
```diff
@@ -162,10 +162,13 @@ The timeout value for all outgoing logs in milliseconds.
 
 **Example:** `export OTEL_EXPORTER_OTLP_LOGS_TIMEOUT=500`
 
-### `OTEL_EXPORTER_OTLP_PROTOCOL`
-
 ## Protocol Configuration
```
Comment (tiffany76) [inline content/en/docs/languages/sdk-configuration/otlp-exporter.md:165]:
```suggestion
## Protocol configuration
```

File: content/en/docs/languages/sdk-configuration/otlp-exporter.md [hunk 3]
```diff
@@ -162,10 +162,13 @@ The timeout value for all outgoing logs in milliseconds.
...
 
+The following environment variables configure the OTLP transport protocol
+an OTLP Exporter uses.
```
Comment (tiffany76) [inline content/en/docs/languages/sdk-configuration/otlp-exporter.md:168]:
```suggestion
an OTLP exporter uses.
```

**Review by tiffany76 — APPROVED** [review]
LGTM with some tiny changes.

**Review by theletterf — APPROVED** [review]

**Pull request merged**